common/static/common/js/discussion/views/discussion_topic_menu_view.js

```diff
@@ -45,7 +45,8 @@
             renderCategoryMap: function(map) {
                 var categoryTemplate = edx.HtmlUtils.template($('#new-post-menu-category-template').html()),
                     entryTemplate = edx.HtmlUtils.template($('#new-post-menu-entry-template').html()),
-                    mappedCategorySnippets = _.map(map.children, function(name) {
+                    mappedCategorySnippets = _.map(map.children, function(nameType) {
```

Comment:
Right now I don't see type used in JavaScript.  However, rather than removing type, would it be more correct to include the same fix?  See the following (untested) code:
```
var type = child[1];
...
if (_.has(map.entries, name) && type == "entry") {
   ...
```

I'm not clear how this is already working without the above.